package.json

```
@@ -92,10 +81,11 @@
     "retext-syntax-mentions": "4.0.0",
     "retext-syntax-urls": "4.0.0",
     "strip-comments": "2.0.1",
-    "stylelint": "16.2.1",
-    "stylelint-config-recess-order": "4.6.0",
-    "stylelint-config-standard-scss": "13.0.0",
     "unified": "11.0.4",
     "vnu-jar": "23.4.11"
+  },
+  "packageManager": "pnpm@8.15.1",
+  "engines": {
+    "node": "^16.13.0"
```

```suggestion
    "node": ">=16.13.0 <=16.20.2"
```

🔗: https://bytearcher.com/articles/semver-explained-why-theres-a-caret-in-my-package-json/

---

same thing?